doc/authoring_command_modules/authoring_commands.md

```diff
@@ -148,8 +148,7 @@ Now the user may identify the target IP config by specifying either the resource
 
 A couple things to note:
 - Currently, `--ids` is not exposed for any command that is called 'create', even if it is configured properly.
-- The supported resource-name values for `id_part` are: `name` and `child_name_{level}`; for example, `child_name_2` is the name of the child two levels under the resource correspond to `name`.
-- Values also available are `child_type_{level}`, `child_namespace_{level}`, and `child_parent_{level}`. Note that these values may not exist for the particular id being parsed.
+- The supported values for `id_part` are `name` and `child_name_X` where X is an integer indicating how far removed from the parent resource the child is.
 
```

Comment:
There are other supported values, that we might include here, (though most of them aren't really being used). 
I did a quick search and no one is using `child_type/namespace/parent_{level}`.
Values like `resource_type/group` are, being used very infrequently.
I don't know how much we care about exposing these, given how rarely they have been used.